docs/api/structures/stream-protocol-response.md

```diff
@@ -0,0 +1,5 @@
+# StreamProtocolResponse Object
+
+* `statusCode` Number - The HTTP response code
+* `headers` Object - A dictionary containing the response headers
```

Comment:
I would suggest replacing "dictionary" with "object", as the term "dictionary" is not really a part of the JS vernacular.

Reply:
:+1: